.pre-commit-config.yaml

```diff
@@ -26,4 +26,3 @@ repos:
     rev: 19.3b0
     hooks:
     - id: black
-      language_version: python3.7
```

Comment:
is this loss of language_versin intentional?

Reply:
Yup, when swapping to Python 3.8 I don't want black to still be forced into using Python 3.7